CHANGELOG.md

```diff
@@ -328,317 +329,317 @@
 - Compress yq binaries ([#748](https://github.com/heroku/heroku-buildpack-nodejs/pull/748))
 - Move etc/vendor to lib/vendor ([#750](https://github.com/heroku/heroku-buildpack-nodejs/pull/750))
 
-## v167 (2020-02-06)
+## [v167] - 2020-02-06
 - Make minor upgrades to gems ([#735](https://github.com/heroku/heroku-buildpack-nodejs/pull/735))
 - Fix typo in failure metadata call ([#737](https://github.com/heroku/heroku-buildpack-nodejs/pull/737))
 
-## v166 (2019-12-16)
+## [v166] - 2019-12-16
 - Add Node 13 metrics plugin ([#731](https://github.com/heroku/heroku-buildpack-nodejs/pull/731), [#732](https://github.com/heroku/heroku-buildpack-nodejs/pull/732))
 - Fix yarn cache feature flag ([#729](https://github.com/heroku/heroku-buildpack-nodejs/pull/729))
 
-## v165 (2019-10-24)
+## [v165] - 2019-10-24
 - Update README ([#725](https://github.com/heroku/heroku-buildpack-nodejs/pull/725))
 - Update default Node version to 12.x ([#724](https://github.com/heroku/heroku-buildpack-nodejs/pull/724))
 
-## v164 (2019-10-17)
+## [v164] - 2019-10-17
 - Avoid issues in environments requiring proxies for all connections (#708)
 - Extend metadata module in preparation for bin/report work (#709)
 - Vendor the buildpack stdlib (#710)
 - Check the status code when listing S3 buckets (#711)
 - Rename script monitoring metadata (#719)
 - Extend bin/report (#718)
 
-## v163 (2019-09-12)
+## [v163] - 2019-09-12
 - Add a minimal version of bin/report. Not user-facing (#700)
 
-## v162 (2019-09-03)
+## [v162] - 2019-09-03
 - Replace broken kb.heroku.com links with help.heroku.com shortlinks (#698)
 
-## v161 (2019-08-15)
+## [v161] - 2019-08-15
 - Expand A/B test of native yarn caching to 50% of apps (#695)
 
-## v160 (2019-08-07)
+## [v160] - 2019-08-07
 - Roll out A/B test of native yarn caching to a small set of apps (#693)
 
-## v159 (2019-08-01)
+## [v159] - 2019-08-01
 - Updates to metadata saved (#689, #690)
 - Add native yarn cache caching behind a flag (#691)
 
-## v158 (2019-07-10)
+## [v158] - 2019-07-10
 - Don't suppress curl errors (#680)
 - Add logging to track yarn workspace usage (#685)
 - Log out number of js, jsx, and ts files (#686)
 
-## v157 (2019-06-17)
+## [v157] - 2019-06-17
 - Prevent `HTTP_PROXY` env var from blocking version-resolution binary
 
-## v156 (2019-06-12)
+## [v156] - 2019-06-12
 - Turn off npm ci experiment
 
-## v155 (2019-06-06)
+## [v155] - 2019-06-06
 - Add metadata for build steps (#677)
 
-## v154 (2019-06-05)
+## [v154] - 2019-06-05
 - Roll out A/B test of npm ci command to 5% of apps (#676)
 - Internal fixes (#674, #673)
 
-## v153 (2019-05-30)
+## [v153] - 2019-05-30
 - Test new A/B testing module (#671, #672)
 
-## v152 (2019-05-28)
+## [v152] - 2019-05-28
 - Replace nodebin usage in hatchet tests (#669)
 - Add ability to list all node or yarn releases (#668)
 
-## v151 (2019-05-28)
+## [v151] - 2019-05-28
```

Comment:
For v151 and older, the markdown link is broken since there is no corresponding link target in the footer of the README. I presume this is since older releases have had their tags renamed (like was done for the Python buildpack). 

I wonder if it would be less confusing to users to have no link at all for the older releases (ie: for v151 and older to remove the `[` and `]`), rather than a broken link?

Or alternatively, the older release links could be made valid by referencing the archived tag name.

The Python buildpack uses a mixture of both approaches (I labelled the last couple of years worth of releases, some of which had an archived tag, but for the releases even older than that didn't add any link or date at all):
https://github.com/heroku/heroku-buildpack-python/blob/35cbe44e19172ac60701002e8f72d5e1a6ae57ba/CHANGELOG.md?plain=1#L573-L585
https://github.com/heroku/heroku-buildpack-python/blob/35cbe44e19172ac60701002e8f72d5e1a6ae57ba/CHANGELOG.md?plain=1#L989-L999

Reply:
Reverted versions < 152 back to their unlinked form. I had done that change before realizing (when I was creating the release links) that everything before that point had it's git tag archived.